### ISSUE TITLE
contact training hyper-parameters

### ISSUE THREAD

**Issue by lidongyv** [hassony2/obman_train#14]
the current training script seems just skip the collisions part. as u say in the paper, the contact lambda is important, can u provide the hyper-parameters to train the network with contact loss?

**Comment by hassony2** [comment]
Hi ! 

Thank you for your interest !
The options to finetune with contact can be found in the file release_models/obman/opt.txt

The relevant part you are asking about is:

```
collision_lambda: 0.3125
collision_mode: dist_tanh
collision_thresh: 20.0
contact_lambda: 0.3125
contact_mode: dist_tanh
contact_target: all
contact_thresh: 10.0
contact_zones: zones
```

Hope this helps !

**Comment by lidongyv** [comment]
thank u so much!
And I just made the code run in multi-GPUs with data-parallel mode, not sure whether the problems come from the pytorch version. The main problem is that the torch.gather() function will raise an error handling the numpy array, so transforming every variable in the forward results to cuda tensor will solve the problem. Just a recommendation for guys running the code in pytorch 1.2.0+ on multi-GPUs env.
